core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -334,6 +337,23 @@ def _fit_single(self, X, y, model_base, use_child_oof, time_limit=None, **kwargs
             self.models = [model_base]
         self._add_child_times_to_bag(model=model_base)
 
+    def _get_default_fold_fitting_strategy(self):
+        try:
+            ray = try_import_ray()
```

Comment:
Does this spam logs if it raises exception for every model it is tried on? If so, we may want to alter so it only logs the install message once

Reply:
I'm not displaying the error message in the exception, so we should be fine.